frontend/packages/operator-lifecycle-manager/src/components/descriptors/reference/reference.md

```diff
@@ -15,12 +15,14 @@
   12. [podAffinity](#12-podaffinity)
   13. [podAntiAffinity](#13-podantiaffinity)
   14. [selector](#14-selector)
-  15. [fieldGroup](#15-fieldgroup)
-  16. [arrayFieldGroup](#16-arrayFieldGroup)
+  15. _DEPRECATED_
+  16. _DEPRECATED_
```

Comment:
I think we should just remove these.
```suggestion
```

Reply:
Actually, so that we don't mess up the numbering, link to the headings in the deprecated section, then keep the #15 an #16 headings below and add a link to the deprectated section as well:

```suggestion
  15. [_DEPRECATED_ fieldGroup](#fieldGroup) 
  16. [_DEPRECATED_ arrayFieldGroup](#arrayFieldGroup)
```